lib/services/access_checker.go

```diff
@@ -470,6 +463,29 @@ func (a *accessChecker) checkAllowedResources(r AccessCheckable) error {
 	return trace.AccessDenied("access to %v denied, not in allowed resource IDs", r.GetKind())
 }
 
+// mapsToURCResource matches requested resource with its respective
+// resource type stored in the unified resource cache.
+func mapsToURCResource(requestedR types.ResourceID, r AccessCheckable) bool {
```

Comment:
I'm not 100% about the name `mapsToURCResource` - but I can't provide an unambiguously better one. How about `matchesUCRResource()`?

Not a blocker.

Reply:
I swear `matchesUCRResource` was what I picked first. 
Now updated in https://github.com/gravitational/teleport/pull/55922/commits/330a8aa55467e9dc3d272772356a1c71ce43bdf6#diff-fde01fb35b1a517643fb3a7ace71c2346d688e90c78e3c3fa5cf51d491ae2649R468